server.js

```diff
@@ -1,16 +1,90 @@
 const express = require('express');
 const helmet = require('helmet');
 const cors = require('cors');
+const mongoose = require('mongoose');
+
+const Friend = require('./Models/FriendModel')
+
+mongoose.connect('mongodb://localhost/friends')
+  .then(() => {
+    console.log('Successfully connected to Mongo DB... you did it; yay!')
+  })
+  .catch(() => {
+    console.log(`You did NOT successfully connect to Mongo DB, make sure it's running; also... don't be like Jake, Jake only likes SQL`)
+  })
 
 const server = express();
 
 server.use(helmet());
 server.use(cors());
 server.use(express.json());
 
+
 server.get('/', (req, res) => {
   res.status(200).json({ api: 'running' });
 });
 
+server.route('/api/friends').get((req, res) => {
+  Friend.find()
+    .then((response) => {
+      res.status(200).json(response)
+    })
+    .catch(() => {
+      res.status(500).json(`{ errorMessage: "The friends information could not be retrieved." }`)
+    })
+})
+
+server.get('/api/friends/:id', (req, res) => {
+  const { id } = req.params
+    Friend.findById(id)
+      .then((response) => {
+        if(!response)
+           res.status(404).json(`{ message: "The friend with the specified ID does not exist." }`)
+        res.status(200).json(response)
+      })
+      .catch((error) => {
+        res.status(500).json(`{ errorMessage: "The friend information could not be retrieved." }`)
```

Comment:
We don't need the backticks around the object `{ errorMessage: "The friend information could not be retrieved." }`, otherwise, you'll be sending that object to the client as a string.